app/src/test/java/com/findmyrecycling/ProductTests.kt

```diff
@@ -88,7 +88,7 @@ class ProductTests {
         assertTrue(allProducts!!.isNotEmpty())
         var containsBatteries = false
         allProducts!!.forEach {
-            if (it.product.equals("Rechargable Batteries") || it.product.equals("Lead Acid Batteries")) {
+            if (it.product == "Rechargable Batteries" || it.product.equals("Lead Acid Batteries")) {
```

Comment:
simplified operator

Reply:
This was covered in a previously submitted review. When that occurs, first in wins.